src/test/java/com/commercetools/sync/products/utils/ProductUpdateActionUtilsTest.java

```diff
@@ -182,6 +239,35 @@ public void buildChangeMasterVariantUpdateAction_changesMasterVariant() throws E
     }
 
     @Test
+    public void buildChangeMasterVariantUpdateAction_withEmptyKey() throws Exception {
```

Comment:
could you please follow the naming convention used for naming tests (it makes it much easier to know everything about the test without looking on the code):

_MethodName_StateUnderTest_ExpectedBehavior_

so in this case it would be `buildVariantsUpdateActions_withEmptyKey_ShouldNotBuildActionAndTriggerCallback`

Reply:
2ee29177